source/guides/recipes/dependency-management.md

```diff
@@ -66,7 +66,7 @@ Enter the development environment, create a new directory, and set up niv with a
 $ nix-shell
 [nix-shell]$ mkdir old
 [nix-shell]$ cd old
-[nix-shell]$ niv init --nixpkgs nixos/nixpkgs --nixpkgs-branch 18.09
+[nix-shell]$ niv init --nixpkgs nixos/nixpkgs --nixpkgs-branch 23.05
```

Comment:
```suggestion
[nix-shell]$ niv init --nixpkgs nixos/nixpkgs --nixpkgs-branch 21.11
```

That's the latest version with an older `hello` version, namely `2.10` as already in the tutorial.